### DISCUSSION TITLE
Add map filtering UI

### DISCUSSION THREAD

**Discussion by timiimit** [ppy/osu#28493] [Ideas]
Currently map filtering by can be done through text (for example *status=loved*, *ar>9*, ...). I think it would make sense to add UI for this to make it more intuitive. Currently there is no way for users to know such filtering is even possible and I personally had to go look at source code when I was looking what is the name of something i wanted to filter.

**Comment by Stoppedpuma** [comment]
Might be apart of the upcoming song select redesign as there's a filter button.
![image](https://github.com/ppy/osu/assets/58333920/345d3783-8b07-4035-97c4-85e3eb732be7)
